src/IIIFPresentation/API/Features/Storage/Helpers/ThumbnailX.cs

```diff
@@ -0,0 +1,33 @@
+﻿using IIIF.Presentation.V3.Content;
+
+namespace API.Features.Storage.Helpers;
+
+public static class ThumbnailX
+{
+    private const int ThumbnailSize = 100;
+    private const int AcceptableDelta = 2;
+    
+    public static string? GetThumbnailPath(this List<Image> thumbnails)
```

Comment:
Worth adding XML comments, or expanding method name to make it clearer that this will get thumbnail closest to X +/- threshold? (may not be required depending on whether changes made for previous comment)